nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -347,3 +352,7 @@ The `commit_default` job field has been renamed to `dryrun_default` and the defa
 
 !!! important
     Nautobot no longer enforces any job behavior when dryrun is set. It is now the job author's responsibility to define and enforce the execution of a "dry run".
+
+### Request Property
+
+The `request` property has been changed to a celery request instead of a Django web request and no longer includes the information from the web request that initiated the Job. The `user` object is now available as `self.user` instead of `self.request.user`.
```

Comment:
```suggestion
The `request` property has been changed to a Celery request instead of a Django web request and no longer includes the information from the web request that initiated the Job. The `user` object is now available as `self.user` instead of `self.request.user`.
```